### PR TITLE
Refactor table component

### PR DESCRIPTION
**Pull request by adrianmroz**

### CODE REVIEW

File: src/client/visualizations/table/highlight.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  top: number;
+  left: number;
+}
+
+export const Highlighter: React.SFC<HighlighterProps> = ({ top, left }) => {
+  return <div className="highlight-cont">
+    <div className="highlight">
+      <div className="highlighter" key="highlight" style={{ top, left }} />
+    </div>
+  </div>;
```
Comment (bigsergey) [inline src/client/visualizations/table/highlight.tsx]: three element to render highlight? Is it possible to optimize dom node number?

File: src/client/visualizations/table/measures-header.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  cellWidth: number;
+  series: ConcreteSeries[];
+  commonSort: Sort;
+  showPrevious: boolean;
+}
+
+export const MeasuresHeader: React.SFC<MeasuresHeaderProps> = props => {
+  const { cellWidth, series, commonSort, showPrevious } = props;
+
+  function sortDirection(series: ConcreteSeries, period = SeriesDerivation.CURRENT): SortDirection | null {
```
Comment (bigsergey) [inline src/client/visualizations/table/measures-header.tsx]: move outside render function

**Comment by adrianmroz** [comment]
> Standart question from me: "Tests?" :-D

Standard answer: nope. I'm ashamed but I'm also tired of this PR :)

File: src/client/visualizations/table/highlight.tsx (hunk 1 above)
Comment (adrianmroz) [reply src/client/visualizations/table/highlight.tsx]: Yeah, seems excessive, but I don't want to touch it there.

**Pull request merged**